source/github-events/on-new-review.ts

```diff
@@ -0,0 +1,13 @@
+import delegate from 'delegate-it';
+
+function callbackWrapper(callback: VoidFunction): (event: CustomEvent) => void {
+	return function (event) {
+		if (/^pull request #\d+ updated$/.test(event.detail.data.reason)) {
+			callback();
+		}
+	};
+}
+
+export default function onNewReviewComment(callback: VoidFunction): void {
+	delegate(document, '.js-pull-refresh-on-pjax', 'socket:message', callbackWrapper(callback), {capture: true});
```

Comment:
This triggers on other events besides new reviews, e.g. when editing or deleting reviews.

In the first case it's actually a good thing I guess, because the contents of the embedded snippets might have been changed and/or added.

Reply:
`socket:message` can't be used I think. The message doesn't always equal DOM change, which sometimes happens later. We had this issue a few years ago when implementing `onNewComments`, not sure if it applies here.

If it works… it's fine for now.